__tests__/unit/specs/mixins/transaction-types.spec.ts

```diff
@@ -48,91 +48,91 @@ describe("Mixins > Transaction Types", () => {
 
       expect(wrapper.vm.isBusinessEntityRegistration(MagistrateTransaction.ENTITY, TypeGroupTransaction.MAGISTRATE, {
         type: MagistrateTransactionEntityType.BUSINESS,
-        subType: MagistrateTransactionEntitySubType.NONE,
+        // subType: MagistrateTransactionEntitySubType.NONE,
```

Comment:
why are these commented? remove them if they are no longer needed

Reply:
There will be new subTypes and so the comments. But if you prefer, i can remove it for now

Reply:
remove them

Reply:
okay :+1: